.circleci/conditional_config.yml

```diff
@@ -203,7 +203,11 @@ jobs:
       - run: *install_circle_cli
       - run:
           name: Setup application
-          command: cd src/api; bundle exec rake dev:prepare assets:precompile RAILS_ENV=test FORCE_EXAMPLE_FILES=1
+          command: |
+            make -C src/api test_rake
+            make -C src/api test_rack
```

Comment:
Could these two `make` checks already be run between `- run: *install_dependencies` and `- *save_bundler_cache`?

Reply:
Because we don't want to spend any time of our CI saving the bundler cache if we will throw an error anyways.

Reply:
It's not wasted as we are going to use the cache. The usual workflow will be

1. depfu updates rack (saves the cache for this bundle)
2. CI fails because no one updated the [rubygem-rack](https://build.opensuse.org/package/show/OBS:Server:Unstable/rubygem-rack) package
3. Someone updates rubygem-rack
4. CI won't fail anymore (and use the cache)

Reply:
Ok, let's try it.